internal/handlers/pg/msg_getlog.go

```diff
@@ -91,14 +91,20 @@ func (h *Handler) MsgGetLog(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 			"Please star us on GitHub: https://github.com/FerretDB/FerretDB.",
 		}
 
-		// TODO https://github.com/FerretDB/FerretDB/issues/1443
 		state := h.StateProvider.Get()
-		if state.Telemetry == nil {
+
+		switch {
+		case state.Telemetry == nil:
 			startupWarnings = append(
 				startupWarnings,
 				"The telemetry state is undecided; the first report will be sent soon. "+
 					"Read more about FerretDB telemetry and how to opt out at https://beacon.ferretdb.io.",
 			)
+		case state.LatestVersion != mv.Version:
+			startupWarnings = append(
+				startupWarnings,
+				fmt.Sprintf("New version available! Latest version: %s; Current version: %s", state.LatestVersion, mv.Version),
```

Comment:
@Fashander, what do you think about this wording when we report that a new version is available?

Reply:
I don't think so, we don't document any update process anywhere, also it depends on the method of installation (docker users will have a different method of updating than package managers users (which would also have a different commands depending on the manager)) WDYT?

Reply:
I agree with Patryk, yeah

Reply:
That makes sense. Thanks.